### PR TITLE
doc: Copyedit upgrading_configuration.rst

### PR DESCRIPTION
**Pull request by amyreye**
- Grammatical and formatting cleanup

Signed-off-by: Reyes, Amy <amy.reyes@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/tutorials/upgrading_configuration.rst [hunk 1]
```diff
@@ -26,23 +26,27 @@ Scenario XML
...
+  release.
 
-   .. code-block:: console
+* ``<out file>`` is the path to the new scenario XML file generated by the
+  script.
 
-      WARNING:root:hv/DEBUG_OPTIONS/some = '1' is discarded
-      INFO:root:vm[1]/os_config/name = 'OS Name': Guest OS names are no longer needed in scenario definitions.
+The script generates the new scenario XML file by migrating as many data in
+``<your scenario XML>`` as possible, based on the latest schema. If there is any
```
Comment (dbkinder) [inline doc/tutorials/upgrading_configuration.rst:43]: You've been using ``data`` as plural previously and in the next sentence, so for consistency:
```suggestion
``<your scenario XML>`` as possible, based on the latest schema. If there are any
```

File: doc/tutorials/upgrading_configuration.rst [hunk 2]
```diff
@@ -52,19 +56,19 @@ info message indicates an intended drop of obsoleted data.
...
 
 Launch XML
 **********
 
-Starting from ACRN v3.0, data in a launch XML are merged into the
-scenario XML. The same upgrader can be used to upgrade a pair of scenario XML
-and launch XML:
+Starting from ACRN v3.0, data in a launch XML are merged into the scenario XML.
+You can use the same upgrader tool to upgrade a pair of scenario XML and launch
+XML:
```
Comment (dbkinder) [inline doc/tutorials/upgrading_configuration.rst:67]:
```suggestion
XML files into the merged format:
```

File: doc/tutorials/upgrading_configuration.rst [hunk 3]
```diff
@@ -52,19 +56,19 @@ info message indicates an intended drop of obsoleted data.
...
 
-   .. code-block:: bash
+.. code-block:: bash
 
-      misc/config_tools/scenario_config/upgrader.py <your scenario XML> --launch <your launch XML> <out file>
+   misc/config_tools/scenario_config/upgrader.py <your scenario XML> --launch <your launch XML> <out file>
 
-The upgrader will then migrate all data in both XMLs and show messages about
+The upgrader migrates all data in both XMLs and shows messages about
```
Comment (dbkinder) [inline doc/tutorials/upgrading_configuration.rst:73]:
```suggestion
The upgrader migrates all data in both XMLs into the ``<out file>`` and shows messages about
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
A few additional tweaks...

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Pull request merged**